src/Pages/SavedNotes/SavedNotes.js

```diff
@@ -200,65 +203,87 @@ function SavedNotes() {
                 </div>
               </>
             ) : (
-              <div className="flex flex-wrap ">
-                {savedNotes.map((item) => (
-                  <div key={item._id}>
-                    <div
-                      className="m-2 savednote-container w-70vw"
-                      style={{ backgroundColor: item.noteColor }}
-                    >
-                      <h2>{item.title}</h2>
-                      <ReactQuill
-                        key={item._id}
-                        theme="snow"
-                        readOnly
-                        value={item.note}
-                      />
-                      <p>created - {item.date}</p>
-                      {item.tags
-                        ? item.tags.map((item) => (
+              <>
+                <div className="flex-center-center w-100per">
+                  <input
+                    type={"search"}
+                    onChange={(e) => setSearchTerm(e.target.value.toString())}
+                    className="p-1 w-40per search-bar"
+                    placeholder="Search notes..."
+                  />
+                </div>
+                <div className="flex flex-wrap my-3">
+                  {savedNotes
+                    .filter(
+                      (item) =>
+                        item.title
+                          .toLowerCase()
+                          .includes(searchTerm.toLowerCase()) ||
+                        item.note
+                          .toLowerCase()
+                          .includes(searchTerm.toLowerCase())
+                    )
+                    .map((item) => (
+                      <div key={item._id}>
+                        <div
+                          className="m-2 savednote-container w-70vw"
+                          style={{ backgroundColor: item.noteColor }}
+                        >
+                          <h2>{item.title}</h2>
+                          <ReactQuill
+                            key={item._id}
+                            theme="snow"
+                            readOnly
+                            value={item.note}
+                          />
+                          <p>created - {item.date}</p>
+                          {item.tags
+                            ? item.tags.map((item) => (
+                                <button
+                                  className="mx-1"
+                                  style={{
+                                    backgroundColor: "orange",
+                                    borderRadius: "2rem",
+                                    border: "2px solid black",
+                                    padding: "0.5rem",
+                                  }}
```

Comment:
I noticed you used classes in most of the places. Is there a reason for inline styling here?

Reply:
thanks for pointing out, I guess I forgot to put it in the CSS file